bin/initial.sh

```diff
@@ -1,5 +1,13 @@
-#!/bin/bash
+#!/usr/bin/env bash
```

Comment:
we don't usually use a bash other than the system one on /bin/bash

Reply:
Yes I know, but it's a public repo so I thought we should keep it universal incase people use it as inspiration for their own setups. Also if we ever deploy on a different OS, it's one less future bug

Reply:
Surprised to find `/usr/bin/env` is in-fact a POSIX standard location, so this is indeed more correct.